src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -34,46 +34,6 @@ extern bool g_demo_mode;
 
 namespace aca_ovs_l2_programmer
 {
-static int aca_set_port_vlan_workitem(const string port_name, uint vlan_id)
```

Comment:
Please bring back this function, we need that in other scenarios.

Reply:
Fixed